www/themes/custom/upd/assets/css/custom.css

```diff
@@ -88,3 +88,103 @@
   border-width: 2px 2px 0;
   border-radius: 5px 5px 0 0;
 }
+
+/* #64697: Fix color contrast accessibility issues. */
+.button, .site-footer__submit,
+.faq-filter-container a.faq-cat-changer, .faq-filter-container a.news-cat-changer,
+.news-filter-container a.faq-cat-changer, .news-filter-container a.news-cat-changer {
+  background-color: #CE3C22;
```

Comment:
I'd suggest to use variables instead of the hex code. For example,

`:root {
  --main-bg-color: #CE3C22;
}
`

It's going to be easier for maintenance

Reply:
I'll make an icebox ticket for that, then we can circle back. 👍

Reply:
https://redmine.codeenigma.net/issues/65243